lib/rest2.js

```diff
@@ -1167,6 +1092,156 @@ class RESTv2 {
       ttl, scope, writePermission
     }, cb)
   }
+
+  /**
+   * Submit new order
+   *
+   * @param {Order} order - models.Order
+   * @param {Method} cb
+   */
+  submitOrder (order, cb) {
+    return this._makeAuthRequest('/auth/w/order/submit', order.toNewOrderPacket(), cb)
+      .then((res) => {
+        const notification = new Notification(res)
+        // 2 orders can be returned if OCO was used. But due to the interface
+        // we can only return one. User should use getOrders instead
+        return notification.notifyInfo[0] || []
+      })
+  }
+
+  /**
+   * Update existing order
+   *
+   * @param {Objet} order - updates to order
+   * @param {Method} cb
+   */
+  updateOrder (changes, cb) {
+    return this._makeAuthRequest('/auth/w/order/update', changes, cb)
+      .then(this._takeNotificationInfo)
+  }
+
+  /**
+   * Cancel existing order
+   *
+   * @param {int} id - order id
+   * @param {Method} cb
+   */
+  cancelOrder (id, cb) {
+    return this._makeAuthRequest('/auth/w/order/cancel', { id }, cb)
+      .then((res) => {
```

Comment:
This should be refactored to use `_takeNotificationInfo` like the other methods.